### ISSUE TITLE
Request to add XIR to StellarTerm

### ISSUE THREAD

**Issue by xirkle** [irisli/stellarterm#186]
Hi Iris! How are you?

When you have a moment, could you add our XIR coin please? Thank you!

1. https://www.xirkle.com/.well-known/stellar.toml
2. https://www.xirkle.com/images/xirkle-logo-icon-clear.png
3. brand color is #FF0000

![xirkle-logo-icon-clear](https://user-images.githubusercontent.com/30379932/36076569-db5ce720-0f2b-11e8-94c2-148e349e95d8.png)

**Comment by irisli** [comment]
Your CORS is not configured properly

**Comment by xirkle** [comment]
http is now redirecting to https (**done**)

For CORS, I've added the following to my .htaccess file.

<Location "/.well-known/stellar.toml">
    Header set Access-Control-Allow-Origin "*"
</Location>

**Comment by irisli** [comment]
To test, run this command:
```
curl -i https://xirkle.com/.well-known/stellar.toml
```

Did you include the closing tag:
```
<Location "/.well-known/stellar.toml">
    Header set Access-Control-Allow-Origin "*"
</Location>
```

**Comment by xirkle** [comment]
I added it to the wrong file and the initial test failed.
My sys admin corrected it for me.

I tested it again on https://www.test-cors.org and it passed.

Here are the results:

Sending HEAD request to https://xirkle.com/.well-known/stellar.toml

Fired XHR event: loadstart
Fired XHR event: readystatechange
Fired XHR event: readystatechange
Fired XHR event: load

XHR status: 200
XHR status text: OK
XHR exposed response headers:
last-modified: Wed, 07 Feb 2018 04:53:08 GMT

Fired XHR event: loadend

Let me know if you have any other questions. Thank you Iris!
